ansible/aicloud

```diff
@@ -1,19 +1,19 @@
 [slurm-node]
-# a256-t4-01.srv.aau.dk is_compute=true|bool ansible_become=yes ansible_host=172.21.212.130
-a256-t4-02.srv.aau.dk is_compute=true|bool ansible_become=yes ansible_host=172.21.212.131
-a256-t4-03.srv.aau.dk is_compute=true|bool ansible_become=yes ansible_host=172.21.212.132
+# a256-t4-01.srv.aau.dk is_compute=true|bool gpu_type=t4 ansible_become=yes ansible_host=172.21.212.130
```

Comment:
Hvordan er det, kan vi evt. bruge det der autodiscovery?

Reply:
jamen der var forskellige muligheder for f.eks at bruge nvidia-smi' toolet men in the end fandt jeg det simplere bare at navngive dem selv; der kan både komme ikke-nvidia gpu'er i fremtiden og man kan også forestille sig a10'ere som ikke er ens hvad angår memory og så videre...

Reply:
OK. Hvis det bare virker med hard-coding, så kan vi nok også leve med det.